ClientUI.Shared/ViewModels/LoginViewModel.Designer.cs

```diff
@@ -0,0 +1,81 @@
+﻿//------------------------------------------------------------------------------

```

Comment:
Since we did switch to JSON localization it might be we no longer need this file anymore.

Reply:
I altered all other translations to json localization too, made sure no other resx is lying around